apis/supervisor/config/v1alpha1/types_federationdomain.go.tmpl

```diff
@@ -58,6 +207,32 @@ type FederationDomainSpec struct {
 	// TLS configures how this FederationDomain is served over Transport Layer Security (TLS).
 	// +optional
 	TLS *FederationDomainTLSSpec `json:"tls,omitempty"`
+
```

Comment:
Not this PR really, but noting that the comment for TLS could be clearer:

```
// TLS configures how this FederationDomain is served over Transport Layer Security (TLS).
```
vs
```
// TLS specifies a secret which will contain Transport Layer Security (TLS) configuration for the FederationDomain.

Reply:
Will update.